tests/file_json5.rs

```diff
@@ -65,6 +66,7 @@ fn test_file() {
 }
 
 #[test]
+#[cfg(not(target_os = "windows"))] // there is something wrong with CR LF I cannot solve
```

Comment:
@up9cloud maybe you have an idea what's wrong here?

Reply:
@matthiasbeyer Well, I guess the error message from crate `pest` (parser for json5-rs) uses different EOL.

@szarykott try this?
```diff
- " --> 2:7\n  |\n2 |   ok: true␊\n  |       ^---\n  |\n  = expected null in {}"
+ " --> 2:7\n  |\n2 |   ok: true␍␊\n  |       ^---\n  |\n  = expected null in {}"
```

see https://github.com/pest-parser/pest/blob/b2c350862f52f3b51f6a32c79727e3dec3a408ad/pest/src/error.rs#L514

Reply:
@matthiasbeyer Well, after review the pest (parser for json5-rs) code base, I guess @szarykott didn't set `core.autocrlf`  to `false`.

Reply:
Code is reverted, I'll try to fit it on my side and rely on GitHub's CI/CD to make sure everything is ok.